addons/storyshots/README.md

```diff
@@ -374,35 +374,26 @@ initStoryshots({
 ```
 
 If you are using enzyme, you need to make sure jest knows how to serialize rendered components.
-You can either pass in a serializer (see below) or specify an enzyme-compatible serializer (like [enzyme-to-json](https://github.com/adriantoine/enzyme-to-json), [jest-serializer-enzyme](https://github.com/rogeliog/jest-serializer-enzyme) etc.) as the default `snapshotSerializer` in your config.
-
-Example for jest config in `package.json`:
-```json
-"devDependencies": {
-    "enzyme-to-json": "^3.2.2"
-},
-"jest": {
-    "snapshotSerializers": [
-      "enzyme-to-json/serializer"
-    ]
-  }
-```
+For that, you can pass an enzyme-compatible snapshotSerializer (like [enzyme-to-json](https://github.com/adriantoine/enzyme-to-json), [jest-serializer-enzyme](https://github.com/rogeliog/jest-serializer-enzyme) etc.) with the `snapshotSerializer` option (see below). 
+
 
-### `serializer`
+### `snapshotSerializers`
```

Comment:
You should probably leave `serializer` in docs, while it's still supported, but mention that is deprecated.

Reply:
done